root/static/scripts/common/components/EntityLink.js

```diff
@@ -191,9 +191,8 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
     href = entity.href_url;
   }
 
-  // TODO: support name variations for all entity types?
-  if (!subPath &&
-      (entity.entityType === 'artist' || entity.entityType === 'recording')) {
+  // URLs are kind of weird and we probably don't care to set this for them
+  if (!subPath && entity.entityType !== 'url') {
```

Comment:
The same probably goes for collection and genre. I would suggest to extend the list of entity types known to have name variation instead of setting name variation by default.

Reply:
My thought was that anything else than url probably should show a name variation if the content is different than the name, but there might be some places we don't want this? For genre for example I'd expect to want to show name variation once we start allowing aliased genres.

Reply:
Alright, no specific `content` is provided for genre and collection for now anyway.